_posts/2018-02-19-tech-radar.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: post
+title:  "Tech Radar"
+date:   2018-02-19 09:30:30 +0100
+categories:
+teaser: Experiment
+excerpt: >
+  A highly configurable system for auto-generating ‘radar’-style displays of project and team datasets such as backlogs.
+---
+ | FT Labs' Tech Radar is a highly configurable system for auto-generating ‘radar’-style displays of project and team datasets such as backlogs.  <BR>  <BR> The code sits [in this repo](https://github.com/ftlabs/tech-radar), and includes [an example dataset](https://raw.githubusercontent.com/ftlabs/tech-radar/master/client/demo.json) which is the source of data for [this fruit and veg example](https://labs.ft.com/tech-radar/?json=https%3A%2F%2Flabs.ft.com%2Ftech-radar%2Fdemo.json&sortcol=tastiness&segment=colour&sorttype=alphabetical&title=FTLabs+Potential+Projects+State&ringcolor=rainbow&proportionalrings=true&noderepulsion=1&nodeattraction=9&quadrant=bottom+left").| [![Example Tech Radar](/assets/uploads/2018/02/example_radar.png)](/assets/uploads/2018/02/example_radar.png) |
+ | | *fruit and veg demo* |
+
+
+
+## Why a radar, and why ~~now~~ then?
+
+It seemed every internal team wanted their own radar-themed view of their projects, and were grumbling about the effort needed to maintain them manually. Greatly inspired by the [Thoughtworks Radar](https://www.thoughtworks.com/radar), we saw an opportunity to build a tool to automate the generation of radar displays, not using Ruby, taking advantage of an in-house service which exports data from Google Spreadsheets as a JSON feed. Each team could maintain (and share) their projects’ details in a spreadsheet, and launch the radar display via a link within the spreadsheet.
```

Comment:
Why "not using Ruby"?

Reply:
so it works as a static page, in-browser, and also on Windows machines